edgexfoundry/edgex-compose
# build: Pin 3rdy party images to patch level

Pinning 3rd party images in prep for next week's code freeze







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Run `make run ds-virtual delayed-start`
Verify all service bootstrapped successfully
Verify services accessible via API Gateway with token
Verify configurations in Consul using ACL token
